src/models/Database.ts

```diff
@@ -56,6 +56,18 @@ export default class Database {
     return db;
   }
 
+  /** Notifies all repositories' observers about changes. */
+  public async notify(): Promise<void[][]> {
```

Comment:
Whenever any update will come, will it notify all repositories. In turn notify all services?

Reply:
Exactly.